packages/web-twig/src/Resources/components/Item/__tests__/__fixtures__/itemDisabled.twig

```diff
@@ -0,0 +1,2 @@
+<Item label="Item label" isDisabled />
+<Item label="Item label" elementType="a" isDisabled />
```

Comment:
can we test the button disabled? Because we added the attribute

Reply:
not sure if i understand, isn't button disabled the first line? Button is default elementType

in the test is this:
```
<button class="Item Item--disabled" disabled aria-selected="false">
  <span class="Item__label">Item label</span>
</button>
```

Reply:
My bad, didn't see it, sorry.